### ISSUE TITLE
Garbage collector error with make_wiki script

### ISSUE THREAD

**Issue by erogol** [RaRe-Technologies/gensim#635]
I call:
````
python -m gensim.scripts.make_wiki trwiki-latest-pages-articles.xml.bz2 .
````

The only difference is I used trwiki dump

Here what I got:
````
Error when tring to find the memory information on the GPU: initialization error
Error freeing device pointer 0x704b88000 (initialization error). Driver report 0 bytes free and 0 bytes total 
CudaNdarray_uninit: error freeing self->devdata. (self=0x7fc542c1e470, self->devata=0x704b88000)
Exception MemoryError: 'error freeing device pointer 0x704b88000 (initialization error)' in 'garbage collection' ignored
Fatal Python error: unexpected exception during garbage collection
````

**Comment by piskvorky** [comment]
Oh wow. Wiki processing has nothing to do with CUDA or GPUs, so I'm not sure what this is.

Do you have Theano installed? Is it installed properly? (gensim does `import theano` in `gensim/corpora/sharded_corpus.py`, but nothing else)

**Comment by erogol** [comment]
yeap theano works fine.

**Comment by fredtcaroli** [comment]
I had the same problem here. What I did to fix was disable the gpu.
For your case, you should run the script like so:
```bash
THEANO_FLAGS=device=cpu python -m gensim.scripts.make_wiki trwiki-latest-pages-articles.xml.bz2 .
```

**Comment by piskvorky** [comment]
@tmylk this is issue https://github.com/RaRe-Technologies/gensim/issues/696#issuecomment-218332228 again -- fixing urgent.

**Comment by tmylk** [comment]
Fixed in #919
